pydotorg/urls.py

```diff
@@ -1,71 +1,72 @@
-from django.conf.urls import include, url, handler404
```

Comment:
*sigh* So GIT is telling me there is no difference except for line 50 between this branch and master. It also says that everything is up-to-date. However, GitHub says something different.

I give up. :-(

@tjguk do you have an ETA for when this could be fixed. I have the BBC breathing down my neck.